test_outputs/connlist/anp_banp_blog_demo_2_explain_output.txt

```diff
@@ -32,15 +32,15 @@ Denied LIST:
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 CONNECTIONS BETWEEN monitoring/mymonitoring[Pod] => bar/mybar[Pod]:
 
-Allowed LIST:
+Allowed list:
 	Allowed TCP:[1234] due to the following policies // rules:
 		EGRESS DIRECTION (Allowed) due to the system default (Allow all)
 		INGRESS DIRECTION (Allowed)
```

Comment:
Change to 
Egress
Ingress
(remove the DIRECTION)

Reply:
done